CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # Changelog
 
+## 0.4.0
+
+### A new, hand-written parser
+
+This release replaces the generated parser with a hand-written recursive descent parser in Ruff.
```

Comment:
We should say something like. "Ruff just got 2x faster" :)

Reply:
Well, it's the _parser_ which got 2x faster, Ruff only about 30-50% ;)

Reply:
"only" 😆